src/components/MultiHopTrade/components/LimitOrder/hooks/useLimitOrders.tsx

```diff
@@ -90,5 +95,24 @@ export const useLimitOrders = () => {
     })
   }, [limitOrdersQuery.currentData, prevLimitOrdersData, toast, translate])
 
-  return limitOrdersQuery
+  const ordersByActionId = useMemo(() => {
+    return (limitOrdersQuery.data ?? []).reduce(
```

Comment:
nitpick: prefer generic in `.reduce<T>()` vs. type assertion below

Reply:
Done in [2712581](https://github.com/shapeshift/web/pull/9626/commits/271258198517d54466a30d742e57de2d7e53f15b)